Sorting Algorithms/Merge Sort Algorithm/Python/MergeSort.py

```diff
@@ -0,0 +1,57 @@
+# ===== ===== =====
+#
+# Theory of Programming
+#
+# Merge Sort Algorithm
+# http://theoryofprogramming.com/2015/01/29/merge-sort-algorithm/
+# GitHub - https://github.com/VamsiSangam/theoryofprogramming
+# Code Contributor - Saurabh Rane (github.com/ranesr)
+#
+# ===== ===== =====
+
+
+def merge_sort(array):
+	# Check if length of array is more than 1
+	if len(array) > 1:
+	
+		# Getting middle element index
+		mid = len(array)//2
+		left_half = array[:mid]
+		right_half = array[mid:]
+		
+		# Merge sorting left half
+		merge_sort(left_half)
+		
+		# Merge sorting right half
+		merge_sort(right_half)
+		
+		i = 0
+		j = 0
+		k = 0
+		
+		# Merge the two halves
```

Comment:
Could put the merge part of the code in a separate function ```merge``` and then put ``` merge(array, left_half, right_half) ``` here?